tsconfig.json

```diff
@@ -22,14 +20,6 @@
     "jsxImportSource": "pota",
 
     "sourceMap": true,
-
-    "types": ["./ts.pota.d.ts", "./ts.jsx.d.ts"]
   },
-  "include": ["src"],
-  "exclude": [
-    "./node_modules/**",
-    "node_modules",
-    "dist",
-    "dist.standalone"
-  ]
```

Comment:
There was no need to exclude these, as it will only `include` what we've defined.